.changes/next-release/bugfix-endpoints-38893.json

```diff
@@ -0,0 +1,5 @@
+{
+  "type": "bugfix",
+  "category": "``endpoints``",
+  "description": "Prefer signature version from service models over endpoints.json."
+}
```

Comment:
Unsure if we want this, as it should be transparent. #2816 didn't have a changelog entry.

Reply:
This is probably a nuance we don't need to highlight. These aren't public interfaces and as long as the change is a no-op for the end user, it's kind of moot.

Reply:
Removed via https://github.com/boto/botocore/pull/3465/commits/0da938819815bf35605d85519752de3c0a2732ef/